lib/ddtrace/ext/priority.rb

```diff
@@ -4,13 +4,15 @@ module Ext
     # Priority is a hint given to the backend so that it knows which traces to reject or kept.
     # In a distributed context, it should be set before any context propagation (fork, RPC calls) to be effective.
     module Priority
-      # Use this to explicitely inform the backend that a trace should be rejected and not stored.
+      # Use this to explicitly inform the backend that a trace should be rejected and not stored.
+      # This includes rules and rate limits configured by the user through the {RuleSampler}.
       USER_REJECT = -1
-      # Used by the builtin sampler to inform the backend that a trace should be rejected and not stored.
+      # Used by the {PrioritySampler} to inform the backend that a trace should be rejected and not stored.
       AUTO_REJECT = 0
-      # Used by the builtin sampler to inform the backend that a trace should be kept and stored.
+      # Used by the {PrioritySampler} to inform the backend that a trace should be kept and stored.
       AUTO_KEEP = 1
-      # Use this to explicitely inform the backend that a trace should be kept and stored.
+      # Use this to explicitly inform the backend that a trace should be kept and stored.
+      # This includes rules and rate limits configured by the user through the {RuleSampler}.
       USER_KEEP = 2
```

Comment:
We mention for all 4 values that as a result of them "the trace should be [kept/rejected]".

If I understood the PR description correctly, would it make sense to say that, for -1 and 2, "trace MUST be [kept/rejected]"?

(I'm using the definitions from <https://datatracker.ietf.org/doc/html/rfc2119>)

Reply:
Makes sense!